source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,306 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn how to query **geospatial data**
+with the MongoDB Java Driver. 
+
+Geospatial data is data tied to a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with the **GeoJSON**
+format. GeoJSON is a geospatial data format. You can use the GeoJSON format to
+store your data, and you must use the GeoJSON format to query your geospatial
+data with MongoDB. 
+
+
+For a concrete reference, GeoJSON data often resembles the following structure:
```

Comment:
question: at this point is there a distinction between "GeoJSON" and "GeoJSON data" that merits a higher word count?

Reply:
Great question. Whenever I was researching for this page, I had trouble understanding what was inside and outside the GeoJSON specification. I personally think saying "GeoJSON data" is helpful here as in my head it says, 

> "here is what a single piece of GeoJSON data looks like. This isn't the GeoJSON specification, but this might give you a sense of what the thing we are talking about will look like in your app".

Also saying "GeoJSON often resembles the following" feels incorrect to me, as GeoJSON has a precise definition, where as "GeoJSON data" can take many forms.

Please let me know if you disagree with this defense of saying "GeoJSON data".